### PR TITLE
Added mobile SDK installation guide

### PR DESCRIPTION
**Pull request by soffokl**

### CODE REVIEW

File: docs/user-guide/mobile-sdk.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+All SDK libraries available as pre-built packages on the GitHub [release page](https://github.com/mysteriumnetwork/node/releases).
+It can be downloaded and installed manually with any preferred way.
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: It can be downloaded and installed manually in your preferred way.

File: docs/user-guide/mobile-sdk.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+But we also support several standard ways for installing SDK using dependency managers.
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: We also support several standard ways of installing the SDK using dependency managers.

File: docs/user-guide/mobile-sdk.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## iOS
+
+It is possible to get iOS SDK library using [Carthage](https://github.com/Carthage/Carthage) dependency manager.
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: It is possible to get the iOS SDK library using [Carthage](https://github.com/Carthage/Carthage) dependency manager.

File: docs/user-guide/mobile-sdk.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+You can follow the [Carthage quick start guide] to get familiar how to use it or just use the following steps:
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: should we add a link here?

Comment (vkuznecovas) [reply docs/user-guide/mobile-sdk.md]: You can follow the [Carthage quick start guide] to get familiar with the tool or perform following steps:

File: docs/user-guide/mobile-sdk.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Carthage should be installed in the system.
+2. `Cartfile` should be created in the same directory where `.xcodeproj` or `.xcworkspac` located.
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: 2. `Cartfile` should be created in the same directory where `.xcodeproj` or `.xcworkspac` is located.

File: docs/user-guide/mobile-sdk.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+3. `Cartfile` should contain the following line: `github "mysteriumnetwork/node" ~> 0.5.0`.
+    - `mysteriumnetwork/node` - is a path to the GitHub [repository](https://github.com/mysteriumnetwork/node).
+    - `0.5.0` - is a desired version of iOS SDK.
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: - `0.5.0` - is the desired version of iOS SDK.

File: docs/user-guide/mobile-sdk.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+4. Execute command to get required library: `carthage update`.
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: Execute the following command to fetch the required library: `carthage update`.

File: docs/user-guide/mobile-sdk.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+We are publishing an Android SDK library to the [Bintray](https://bintray.com/) repository. And as a result, it is available via [JCenter](https://bintray.com/bintray/jcenter) repository and can be easily imported into your project.
+
+The Android SDK library available at [https://bintray.com/mysterium/Node/mobile-node](https://bintray.com/mysterium/Node/mobile-node)
+
+All you need to do is just add the following line to your `build.gradle` file and resync it: `implementation 'mysterium.network:mobile-node:0.5-rc'`
+
+---
+
+> *It's also possible to install Android SDK library manually following official [guide](https://developer.android.com/studio/projects/android-library#AddDependency)*
```
Comment (vkuznecovas) [inline docs/user-guide/mobile-sdk.md]: > *It's also possible to install Android SDK library manually following the official [guide](https://developer.android.com/studio/projects/android-library#AddDependency)*

**Review by vkuznecovas — COMMENTED** [review]
minor spelling/styling issues

**Review by Arrnas — DISMISSED** [review]

**Comment by soffokl** [comment]
@vkuznecovas all comments fixed.

**Review by tadovas — APPROVED** [review]

**Pull request merged**